pymc3/distributions/continuous.py

```diff
@@ -16,7 +16,7 @@
 from pymc3.theanof import floatX
 from . import transforms
 
-from .dist_math import bound, logpow, gammaln, betaln, std_cdf, i0, i1, alltrue_elemwise, DifferentiableSplineWrapper
+from .dist_math import bound, logpow, gammaln, betaln, std_cdf, i0, i1, alltrue_elemwise, SplineWrapper
```

Comment:
I think this line is too long, we can wrap this import in brackets and do multiline import